### PR TITLE
add n30793 notes and questions

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

**Comment by glozow** [comment]
cc @tdb3 :)

File: _posts/2024-10-02-#30793.md [hunk 1]
```diff
@@ -10,13 +10,53 @@ status: upcoming
...
+
+1. If an RPC has a non-string argument, does anything special need to be done to handle it?
+
+1. What specifically does it mean that the RPC is
+   ["hidden"](https://github.com/bitcoin-core-review-club/bitcoin/commit/8ec094959dc6afd46a709190d2deb58a50593fb7#diff-9c5b83de6dc84af277e352c88b9291aa44340a3c75f572a0b51661eb0a838de9R1131)? Why `hidden` and not `net`?
+
+1. Why can't we just directly access the orphanage from the RPC code? Why don't we just add a
+   PeerManager function that returns a reference to the `TxOrphanage`, which would be more extensible?
+
+1. What is the maximum size of the result from this RPC? Is there a limit to how many orphans are retained?
```
Comment (tdb3) [inline _posts/2024-10-02-#30793.md:48]: Could add a hint here that aids the reader in later answering "able to tell how long a transaction has been in the orphanage".  Maybe:

```diff
- 1. What is the maximum size of the result from this RPC? Is there a limit to how many orphans are retained?
+ 1. What is the maximum size of the result from this RPC? Is there a limit to how many orphans are retained? Is there a limit to how much time orphans are retained?
```

File: _posts/2024-10-02-#30793.md [hunk 2]
```diff
@@ -10,13 +10,53 @@ status: upcoming
...
+
+    - Its purpose is similar to that of `getrawaddrman`. Most likely, developers will be the main users.
+
+## Questions
+
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
+
+1. What is an orphan transaction? At what point do transactions enter the orphanage (can you find the code)?
+
+1. What command can you run to get the list of available RPCs?
```
Comment (tdb3) [inline _posts/2024-10-02-#30793.md:33]: Maybe too obvious, but potentially helpful to help prime for the later `hidden` question.

```diff
- 1. What command can you run to get the list of available RPCs?
+ 1. What command can you run to get the list of available RPCs? Why don't you see `getorphantxs` in the list?
```

**Review by tdb3 — COMMENTED** [review]
These look great, and incorporate the suggestions from IRC nicely!
Left a couple of very minor things.  Feel free to ignore.

File: _posts/2024-10-02-#30793.md (hunk 1 above)
Comment (glozow) [reply _posts/2024-10-02-#30793.md]: Sure, added

File: _posts/2024-10-02-#30793.md (hunk 2 above)
Comment (glozow) [reply _posts/2024-10-02-#30793.md]: Gonna keep this one quiet 🤭 maybe can be a hint if people don't get the later question

**Pull request merged**